redi-connect-front/src/config/config.tsx

```diff
@@ -42,7 +42,7 @@ if ((process.env.REACT_APP_REDI_LOCATION as RediLocation) === 'berlin') {
     { id: 'findingInternship', label: 'Finding an internship', group: 'careerSupport' },
     { id: 'freelancing', label: 'Freelancing', group: 'other' },
     { id: 'salesforce', label: 'Salesforce', group: 'other' },
-    { id: 'dontKnowYet', label: "I don't know yet", group: 'other' }
+    { id: 'iDontKnowYet', label: "I don't know yet", group: 'other' }
```

Comment:
Reverted and deleting `compact()` solved the problem indeed .What I noticed: it wasn't a problem for Munich, only for Berlin. Is that something we should keep in mind / worry about?

Reply:
No. The "bug" actually appeared when we introduced Munich categories but forgot to do `compact()` for Munich's "I don't know" category. But it wasn't a bug after all, since the form validation specs have changed and we forgot to remove compact.
Long story short: we're all good now.